core/types/calls.cc

```diff
@@ -249,7 +249,7 @@ SymbolRef guessOverload(const GlobalState &gs, SymbolRef inClass, SymbolRef prim
                 return true;
             }
             if (getArity(gs, s1) == getArity(gs, s2)) {
-                return s1._id < s2._id;
+                return s1.rawId() < s2.rawId();
```

Comment:
It's a small thing, but because the kind bits are stored in the lower three bits, this comparison isn't consistent for symbol refs of different kinds; it's not always the case that an arbitrary class ref and arbitrary method ref compare the same way. I'm not sure if this is a useful distinction to have, but it seems nice to partition the space of ids such that they don't interleave.

Reply:
That's a good point. I'm amenable to changing where kind is stored.

Reply:
Done.